runtime/compiler/env/VMJ9.cpp

```diff
@@ -5043,6 +5043,100 @@ TR_J9VMBase::getMemberNameFieldKnotIndexFromMethodHandleKnotIndex(TR::Compilatio
    return knot->getOrCreateIndex(mnObject);
    }
 
+TR::KnownObjectTable::Index
+TR_J9VMBase::getMethodHandleTableEntryIndex(TR::Compilation *comp, TR::KnownObjectTable::Index vhIndex, TR::KnownObjectTable::Index adIndex)
```

Comment:
I just want to add that "method handle table entry index" is pleasantly trochaic :slightly_smiling_face:

Reply:
and it's a pentameter no less 😛